### PR TITLE
Boris/add tim javascript

### PR DESCRIPTION
**Pull request by borisjaskerovich**

### CODE REVIEW

File: public/js/amazonpolly-public.js [hunk 1]
```diff
@@ -17,3 +17,18 @@
...
 
 })( jQuery );
+
+function start_tim_limitless() {
+	getTimDemands();
+}
+
+function getTimDemands() {
+    var page_url = window.location.href;
+    var api = "https://mediamart.tv/sas/player/wordpressPluginApi.php?page_url="+page_url;
```
Comment (royasoolin) [inline public/js/amazonpolly-public.js]: campaign?

Comment (shlomisut) [reply public/js/amazonpolly-public.js]: note not to merge code with `mediamart`

Comment (borisjaskerovich) [reply public/js/amazonpolly-public.js]: i guess i will have the campaign later when we will develop the handshake.

Comment (shlomisut) [reply public/js/amazonpolly-public.js]: I think we should start with the handshake, no? isn't it the basis to all ?

Comment (borisjaskerovich) [reply public/js/amazonpolly-public.js]: @royasoolin

File: public/class-amazonpolly-public.php [hunk 2]
```diff
@@ -290,7 +290,7 @@ private function include_audio_player( $language_code, $audio_location, $autopla
 			</audio>
 		</div>';
 		if($common->is_tim_limitless_enabled()){
-            $response .= '<script>start_tim_limitless();</script>';
+            $response .= '<script src="https://mediamart.tv/sas/ads/prod/js/amazon_plugin/startup.js"></script>';
```
Comment (shlomisut) [inline public/class-amazonpolly-public.php]: script should not be a `js` - we shoudl be able to control it - so `php` it is

**Review by royasoolin — APPROVED** [review]

**Pull request merged**